packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -156,6 +156,13 @@ export interface CommonAutoScalingGroupProps {
    * @default none
    */
   readonly spotPrice?: string;
+
+  /**
+   * Configuration for health checks
+   *
+   * @default - HealthCheckConfiguration with defaults.
```

Comment:
If `healthCheck` is not specified, then there are no health checks, right? Fix the default description.

Reply:
I think the health check is mandatory. The [CloudFormation documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-group.html#cfn-as-group-healthchecktype) says EC2 is the default, and the [API reference](https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_AutoScalingGroup.html) says the value is required.

Reply:
Got it, so we should say that EC2 with no grace period is the default.

Reply:
Done